iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ThreeWindingsTransformer.java

```
@@ -319,6 +319,19 @@ default Substation getNullableSubstation() {
      */
     Leg getLeg3();
 
+    default Leg getLeg(Side side) {
+        switch (side) {
+            case ONE:
+                return getLeg1();
+            case TWO:
+                return getLeg2();
+            case THREE:
+                return getLeg3();
+            default:
+                return null;
```

After a second thought, can you better throw here instead of returing null.